### ISSUE TITLE
File won't open when a search is present

### ISSUE THREAD

**Issue by svendahlstrand** [ridiculousfish/HexFiend#224]
If a search phrase is present when opening a file, it won't actually open. Steps to reproduce:

1. Open Hex Fiend (without opening a file)

2. Search for anything (like `Cmd` + `f` hello)

3. Open any file (`File` > `Open File...` or drag and drop)

The file won't open, and the search UI disappears (see screenshot below). Hex Fiend 2.12 on macOS 10.15.3.

```
error 09:04:30.525264+0100  Hex Fiend *** Collection <__NSArrayM: 0x600003467210> was mutated while being enumerated.
error 09:04:30.529129+0100  Hex Fiend (
  0   CoreFoundation                      0x00007fff38b588ab __exceptionPreprocess + 250
  1   libobjc.A.dylib                     0x00007fff6ee12805 objc_exception_throw + 48
  2   CoreFoundation                      0x00007fff38bd13ca __NSFastEnumerationMutationHandler + 159
  3   Hex Fiend                           0x000000010ec63262 -[BaseDataDocument adoptWindowController:fromTransientDocument:] + 398
  4   Hex Fiend                           0x000000010ec7d3a8 -[MyDocumentController replaceTransientDocument:] + 478
  5   Hex Fiend                           0x000000010ec7d65a __80-[MyDocumentController openDocumentWithContentsOfURL:display:completionHandler:]_block_invoke + 212
  6   AppKit                              0x00007fff3624b658 __80-[NSDocumentController openDocumentWithContentsOfURL:display:completionHandler:]_block_invoke.906 + 183
  7   AppKit                              0x00007fff362572bf ___NSMainRunLoopPerformBlockInModes_block_invoke + 25
  8   CoreFoundation                      0x00007fff38adc7ab __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ + 12
  9   CoreFoundation                      0x00007fff38adc6ed __CFRunLoopDoBlocks + 379
  10  CoreFoundation                      0x00007fff38adb731 __CFRunLoopRun + 1257
  11  CoreFoundation                      0x00007fff38adabd3 CFRunLoopRunSpecific + 499
  12  HIToolbox                           0x00007fff3763065d RunCurrentEventLoopInMode + 292
  13  HIToolbox                           0x00007fff3763039d ReceiveNextEventCommon + 600
  14  HIToolbox                           0x00007fff37630127 _BlockUntilNextEventMatchingListInModeWithFilter + 64
  15  AppKit                              0x00007fff35ca0ba4 _DPSNextEvent + 990
  16  AppKit                              0x00007fff35c9f380 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1352
  17  AppKit                              0x00007fff35c9109e -[NSApplication run] + 658
  18  AppKit                              0x00007fff35c63465 NSApplicationMain + 777
  19  libdyld.dylib                       0x00007fff701807fd start + 1
  20  ???                                 0x0000000000000001 0x0 + 1
)
```

![example](https://user-images.githubusercontent.com/35639/74514757-856e7800-4f0d-11ea-8a5b-601efc3c6caf.png)

**Comment by kainjow** [comment]
Wow, nice find. This looks like a 9 year old bug 😲! Should be easy to fix.

**Comment by kainjow** [comment]
Nice find! Thanks for reporting, I'll look into a fix.

**Comment by kainjow** [comment]
Fixed. Oddly enough, the property for subviews is [declared](https://developer.apple.com/documentation/appkit/nsview/1483539-subviews?language=objc) as returning a copy, so this shouldn't be necessary. But most likely they are implementing this as a method and not default property implementation (NSView is old), so it's possibly not actually returning a copy. I was able to fix by iterating over a copy explicitly, but made it simpler by just setting the subviews to nothing.
